presto-orc/src/main/java/com/facebook/presto/orc/OrcSelectiveRecordReader.java

```diff
@@ -354,6 +355,7 @@ public Page getNextPage()
             }
 
             positionsToRead = streamReader.getReadPositions();
+            verify(positionCount == 1 || positionsToRead[positionCount - 1] - positionsToRead[0] >= positionCount, "Positions must monotonically increase");
```

Comment:
@oerling actually, the difference probably needs to be >= positionCount - 1, not positionCount